tests/contrib/flask/test_flask_snapshot.py

```diff
@@ -92,13 +93,17 @@ def flask_client(flask_command, flask_env, flask_port):
         proc.wait()
 
 
-@pytest.mark.snapshot(ignores=["meta.flask.version"])
+@pytest.mark.snapshot(
+    ignores=["meta.flask.version"], variants={"220": "2.2.0" == flask_version, "": "2.2.0" != flask_version}
```

Comment:
these are going to fail as soon as `2.2.1`, or `2.3.0` are released

Reply:
I just changed that to use `ddtrace.contrib.flask.patch.flask_version`, I hope there is no import loop or anything as it is only used in tests